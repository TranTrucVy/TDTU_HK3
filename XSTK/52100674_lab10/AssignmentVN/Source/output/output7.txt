Winifred Mckeown -> Clive Sinclair -> Jayden-James Mccullough
This id not exist
Winifred Mckeown -> Clive Sinclair -> Jayden-James Mccullough
Winifred Mckeown -> Clive Sinclair -> Jayden-James Mccullough
Winifred Mckeown -> Clive Sinclair -> Jayden-James Mccullough
Winifred Mckeown -> Clive Sinclair -> Jayden-James Mccullough
Jayden-James Mccullough
Jayden-James Mccullough -> Clive Sinclair
Jayden-James Mccullough -> Clive Sinclair
Daniyal Bellamy -> Clive Sinclair -> Jacob Christensen -> Jayden-James Mccullough -> Khia Keller -> Ziggy Reilly -> Shanaya Lester -> Maisie Worthington
Jayden-James Mccullough -> Daniyal Bellamy -> Clive Sinclair -> Anita Mansell -> Jacob Christensen -> Khia Keller -> Axl Medrano -> Ziggy Reilly -> Shanaya Lester -> Maisie Worthington -> Simeon Hooper
This id not exist
