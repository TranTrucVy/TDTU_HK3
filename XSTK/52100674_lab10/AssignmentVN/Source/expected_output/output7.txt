Ziggy Reilly -> Jacob Christensen -> Teddy Neal -> Anita Mansell -> Clive Sinclair -> Axl Medrano -> Winifred Mckeown -> Jayden-James Mccullough -> Maisie Worthington -> Simeon Hooper
This id not exist
Ziggy Reilly -> Jacob Christensen -> Teddy Neal -> Anita Mansell -> Clive Sinclair -> Axl Medrano -> Winifred Mckeown -> Jayden-James Mccullough -> Maisie Worthington -> Simeon Hooper
Ziggy Reilly -> Jacob Christensen -> Teddy Neal -> Clive Sinclair -> Axl Medrano -> Winifred Mckeown -> Jayden-James Mccullough -> Maisie Worthington -> Simeon Hooper
Jacob Christensen -> Teddy Neal -> Clive Sinclair -> Ziggy Reilly -> Axl Medrano -> Winifred Mckeown -> Jayden-James Mccullough -> Simeon Hooper
Jacob Christensen -> Teddy Neal -> Clive Sinclair -> Ziggy Reilly -> Axl Medrano -> Winifred Mckeown -> Jayden-James Mccullough -> Simeon Hooper
Ziggy Reilly -> Jacob Christensen -> Teddy Neal -> Anita Mansell -> Clive Sinclair -> Axl Medrano -> Winifred Mckeown -> Jayden-James Mccullough -> Maisie Worthington -> Simeon Hooper
Ziggy Reilly -> Jacob Christensen -> Teddy Neal -> Clive Sinclair -> Axl Medrano -> Winifred Mckeown -> Jayden-James Mccullough -> Maisie Worthington -> Simeon Hooper
Ziggy Reilly -> Jacob Christensen -> Teddy Neal -> Clive Sinclair -> Axl Medrano -> Jayden-James Mccullough -> Maisie Worthington -> Simeon Hooper
Ziggy Reilly -> Jacob Christensen -> Teddy Neal -> Clive Sinclair -> Axl Medrano -> Winifred Mckeown -> Jayden-James Mccullough -> Maisie Worthington -> Simeon Hooper
Jayden-James Mccullough -> Jacob Christensen -> Teddy Neal -> Clive Sinclair -> Axl Medrano -> Daniyal Bellamy -> Ziggy Reilly -> Shanaya Lester -> Maisie Worthington -> Simeon Hooper
Jayden-James Mccullough -> Jacob Christensen -> Teddy Neal -> Clive Sinclair -> Siana Nicholson -> Daniyal Bellamy -> Axl Medrano -> Ziggy Reilly -> Shanaya Lester -> Maisie Worthington -> Simeon Hooper
This id not exist
Shanaya Lester -> Jacob Christensen -> Teddy Neal -> Clive Sinclair -> Siana Nicholson -> Daniyal Bellamy -> Axl Medrano -> Ziggy Reilly -> Maisie Worthington -> Simeon Hooper
Jayden-James Mccullough -> Jacob Christensen -> Teddy Neal -> Clive Sinclair -> Siana Nicholson -> Daniyal Bellamy -> Axl Medrano -> Ziggy Reilly -> Shanaya Lester -> Maisie Worthington -> Simeon Hooper
77,Jayden-James Mccullough,92
Jayden-James Mccullough -> Jacob Christensen -> Teddy Neal -> Clive Sinclair -> Siana Nicholson -> Daniyal Bellamy -> Simeon Hooper -> Shanaya Lester -> Maisie Worthington
Simeon Hooper -> Teddy Neal -> Jacob Christensen -> Maisie Worthington -> Clive Sinclair -> Shanaya Lester -> Daniyal Bellamy -> Siana Nicholson -> Jayden-James Mccullough
